.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
Would a shell session dump be more appropriate?

Reply:
Something like `scriptcs/lint >> lint_errors.txt`

Reply:
Something like scripts/lint >> lint_errors.txt

Reply:
Meh, we can just leave it with the screenshot. I can't think of a concise way of telling people to copy/paste a command and its output without it sounding condescending.